utils/taskcluster_downloader.py

```diff
@@ -48,7 +50,7 @@ def donwload_logs(group_id, output):
     group = queue.listTaskGroup(group_id)
 
     for task in group["tasks"]:
-        if task["status"]["state"] != "completed":
+        if task["status"]["state"] != "completed" and task["status"]["state"] != "running":
```

Comment:
nit (non-blocking): `if task["status"]["state"] not in ("completed", "running"):` would be easier to read.